documents/charter.md

```diff
@@ -16,6 +16,13 @@ Application providers are even designing algorithms to detect the presence of su
 Instead, it would be beneficial, for both the application provider and network operator, to signal network attributes to the application to self-adapt its video traffic to conform to the specified characteristics.
 The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
 
+L4S is a recently standardized mechanism where nodes in the network assist endpoints in discovering optimal sending rates through continuous ECN feedback, interpreted by purpose-built congestion control algorithms. 
+However, L4S alone is not sufficient to communicate network properties related to video self-adaptation. 
+L4S and ECN operate on RTT timescales, reflecting the current state of a network buffer, whereas the network properties related to video traffic reflect policies that tend to be stable over orders of magnitude longer timescales. 
+It is possible to construct an L4S-capable traffic policer that marks packets that do not conform to some policy instead of dropping or queueing them. 
+Such a policer is not explicitly distinguishable from a network node that experiences congestion. 
```

Comment:
```suggestion
While it is possible to construct an L4S-capable traffic policer that marks packets that do not conform to some policy instead of dropping or queueing them, such a policer does not empower a video receiver to adapt the resources it requests. 
The receiver is limited to transport-level adaptation, rather than application-level adaptation, and cannot do more than it does when a network node along the path between sender and receiver experiences congestion. 
```

Reply:
It may be worth pointing out that ECN and L4S use IP header fields that may be rewritten at network boundaries, but I'll let you Do The Right Thing.

Reply:
That is a good point, but I'm worried that we'll inflate the charter text if we start listing all kinds of potential issues.